packages/plugins/experimental/plugin-script/src/compiler/compiler.ts

```diff
@@ -208,3 +217,36 @@ const analyzeSourceFileImports = (code: string): ParsedImport[] => {
     };
   });
 };
+
+const httpPlugin: Plugin = {
+  name: 'http',
+  setup: (build) => {
+    // Intercept import paths starting with "http:" and "https:" so
```

Comment:
If these are your comments -- go to 120 chars (prefer where poss 1 sentence per line).